wgpu-types/src/lib.rs

```diff
@@ -593,6 +607,32 @@ impl Default for Limits {
     }
 }
 
+impl Limits {
+    /// These default limits are guarenteed to be compatible with GLES3, WebGL, and D3D11
+    pub fn downlevel_limits() -> Self {
+        Self {
```

Comment:
would be good to add a test in wgpu-core comparing this to the default limit and ensuring the order, but this isn't a blocker for landing